SU2_CFD/src/solvers/CNSSolver.cpp

```diff
@@ -2058,3 +2102,49 @@ void CNSSolver::SetTauWall_WF(CGeometry *geometry, CSolver **solver_container, C
   }
 
 }
+
+void CNSSolver::Setmut_LES(CGeometry *geometry, CSolver **solver_container, CConfig *config) {
+
+  unsigned long iPoint;
+  su2double Grad_Vel[3][3] = {{0.0,0.0,0.0},{0.0,0.0,0.0},{0.0,0.0,0.0}};
+  su2double lenScale, muTurb, rho;
```

Comment:
I would declare all of these down in the loop when you first access the variable and make them const (everything except Grad_Vel, I suppose)